packages/mux-player/src/helpers.ts

```diff
@@ -16,20 +16,19 @@ export const getPlayerVersion = () => player_version;
 
 export const getSrcFromPlaybackId = (
   playbackId?: string,
-  { token, domain = MUX_VIDEO_DOMAIN }: { token?: string; domain?: string } = {}
+  { maxResolution, token, domain = MUX_VIDEO_DOMAIN }: { maxResolution?: string; token?: string; domain?: string } = {}
```

Comment:
suggestion(non-blocking): If `maxResolution` is going to be a set of string literals, either now or as a followup, we should probably type it as such (See `stream-type`/`streamType` and corresponding maps + typings for an example: https://github.com/muxinc/elements/blob/main/packages/mux-video/src/index.ts#L327-L341 & https://github.com/muxinc/elements/blob/main/packages/playback-core/src/types.ts#L39-L55)